web_src/js/index.js

```diff
@@ -729,9 +729,45 @@ function initRepository() {
       return false;
     });
 
+    // Issue/PR Context Menus
+    $('.context-dropdown').dropdown({
+      action: 'hide'
+    });
+
+    // Quote reply
+    $('.quote-reply').click(function (event) {
+      $(this).closest('.dropdown').find('.menu').toggle('visible');
+      const target = $(this).data('target');
+
+      let $content;
+      if ($(this).hasClass('quote-reply-diff')) {
+        const $parent = $(this).closest('.comment-code-cloud');
+        $parent.find('button.comment-form-reply').click();
+        $content = $parent.find('[name="content"]');
+      } else {
+        $content = $('#content');
+      }
+
+      let content = $(`#comment-${target}`).text();
+      const parts = content.split('\n');
+      for (let i = 0; i < parts.length; i++) {
+        parts[i] = `> ${parts[i]}`;
+      }
+      content = `${parts.join('\n')}\n\n`;
+
+      if ($content.val() !== '') {
+        $content.val(`${$content.val()}\n\n${content}`);
+      } else {
+        $content.val(`${content}`);
+      }
+      $content.focus();
+      event.preventDefault();
+    });
+
     // Edit issue or comment content
-    $('.edit-content').click(function () {
-      const $segment = $(this).parent().parent().parent()
+    $('.edit-content').click(function (event) {
+      $(this).closest('.dropdown').find('.menu').toggle('visible');
+      const $segment = $(this).parent().parent().parent().parent()
```

Comment:
Some family tree! 😆

Reply:
Ah, yes this was here before I got here. I can see if there is a similar way to shorten it up with `.closest()`

Reply:
Done.